rpcs3/Emu/RSX/color_utils.h

```diff
@@ -0,0 +1,245 @@
+#pragma once
+
+#include <util/types.hpp>
+#include <Utilities/geometry.h>
+#include "gcm_enums.h"
+
+namespace rsx
+{
+	struct texture_channel_remap_t
+	{
+		u32 encoded;

```

Comment:
no defaults?

Reply:
I can make default 0 to make compilers happy but that's actually an invalid encoding.